### DISCUSSION TITLE
Overload is deprecated. Examples for migration?

### DISCUSSION THREAD

**Discussion by nkabir** [coady/multimethod#119] [Q&A] [answered]
The documentation indicates that `overload` has been deprecated. Is there sample code that demonstrates how implementations that use `overload` should be re-written?

**Comment by coady** [comment, marked as answer]
Added more examples in 2372ea6.

**Comment by nkabir** [comment]
Thank you. In terms of features (rather than performance), is keyword support the primary difference between multimethod and multidispatch? So anyone that used overload should switch to multidispatch enhanced with the parametric implementations?

**Reply by coady** [reply]
Yes, that's the difference. `multidispatch` supports dispatching on keyword arguments.
